radicle-node/src/service/session.rs

```diff
@@ -132,6 +134,7 @@ impl Error {
     pub fn is_transient(&self) -> bool {
         match self {
             Self::InvalidTimestamp(_) => false,
+            Self::ProtocolMismatch => true,
```

Comment:
How come a protocol mismatch is considered transient?

Reply:
Keep in mind this is a mismatch between the git and gossip protocols.

Reply:
Maybe a better naming of the error or the error's comment/description?  By the naming alone, it can be confused with incompatible protocol versions, or a mismatch with other protocols (connecting to SSH somehow).  Also, in the later case the error is persistent (not transient).

Reply:
Yeah, can add a doc comment :+1: